test/RelayServer.test.js

```diff
@@ -48,14 +50,13 @@ contract('RelayServer', function (accounts) {
   let ethereumNodeUrl
   let _web3
   let id
-  let serverError
   let encodedFunction
   let relayClient
   let options, options2
   let keyManager
 
   before(async function () {
-    ethereumNodeUrl = web3.currentProvider.host
+    ethereumNodeUrl = web3.currentProvider.host || 'http://localhost:8545'
```

Comment:
Why? when is it ever not set?

Reply:
leftover, removed.
 (with buidler, the provider is different. alas, there's no standard API for getting a URL, e.g. in websocket there's no "url"